engine/access/rpc/backend/backend_accounts.go

```diff
@@ -93,3 +110,36 @@ func (b *backendAccounts) getAccountAtBlockID(
 
 	return account, nil
 }
+
+func getAccountError(err error) error {
+	errStatus, _ := status.FromError(err)
```

Comment:
Is this functions till going to work with the Multi Error?

Reply:
added code in `getAccountFromAnyExeNode` to wrap all errors in `codes.NotFound` if all of them are `notfound` errors otherwise mark all as `internal` error